src/pages/EventsCalendar/style.scss

```diff
@@ -70,6 +70,18 @@
         font-feature-settings: 'liga';
       }
     }
+
+    &--loading {
+      .q-date__calendar-item {
+        .q-btn--unelevated::before {
+          content: 'refresh';
+          font-size: 0.8rem;
+          top: 60%;
+          animation: spinning 2s infinite;
```

Comment:
Seems like quasar have a loading animation

https://quasar.dev/quasar-plugins/loading

What about using it?

Reply:
Interesting to know. But in this case I don't think it can be applied since it locks the whole screen, which I don't think was necessary in this case. The current animation is more intuitive about what's loading

Reply:
What about this one?
https://quasar.dev/vue-components/inner-loading

Reply:
That looks better, maybe the designer should decide which is more favorable

![lastrecord](https://user-images.githubusercontent.com/66505715/189921444-e28e500a-e079-4533-8c09-5049a9018fb7.gif)

This is how it looks currently